toolkit/locales/en-US/chrome/global/aboutSupport.ftl

```diff
@@ -45,6 +65,13 @@ aboutSupport-appBasicsMultiProcessSupport = Multiprocess Windows
 aboutSupport-appBasicsSafeMode = Safe Mode
 aboutSupport-appBasicsProfiles = Profiles
 
+# This string displays the number of open multiprocess (e10s) windows compared
+# to the overall number of open windows, and the current status of e10s in the
+# browser.
+# $remote (Number) - Number of multiprocess windows
+# $total (Number)  - Total number of open windows
+# $status (String) - Current status of e10s
```

Comment:
That's a very good point. In this case I would write *Number* without any doubt about creating confusion, in both cases
```
$num (Number) - Total number of unread emails
```
Also because some locales could need to use the variable inside the string. 

But that's not the case for `$status`. I guess the only solution would be to have something like this.
```
# $status (Number) - Numerical index used to identify the current status of e10s. The status message corresponding to the index will be displayed in the UI, e.g. “Enabled by user” for 0.
```